index.ts

```diff
@@ -230,10 +226,36 @@ export type Stable64GrowResult = Variant<{
     err: StableMemoryError;
 }>;
 
+/**
+ * A decorator for annotating query methods on external canisters. Can only be
```

Comment:
A decorator should probably `decorate` and not `annotate`. It's especially important to make the distinction because we're calling our custom `$query` and `$update` things annotations